velox/connectors/Connector.h

```diff
@@ -22,6 +22,7 @@
 #include "velox/common/config/SpillConfig.h"
 #include "velox/common/future/VeloxPromise.h"
 #include "velox/core/ExpressionEvaluator.h"
+#include "velox/exec/Spill.h"
```

Comment:
Connector(velox_connector) can't depend on velox_exec, otherwise it would have circular dependency: https://github.com/facebookincubator/velox/blob/bca2c2ce172ff4df4e8bde7b8c9a9420e38720f6/velox/exec/CMakeLists.txt#L88-L93